api/routes.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+from datetime import datetime
+from typing import List
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from api.crud import (
+    get_cities,
+    create_city,
+    get_city_by_id,
+    update_city,
+    delete_city,
+    get_temperatures,
+    get_temperatures_by_city_id,
+    fetch_city_temperature,
+)
+from api.database import get_db
+from api.models import Temperature, City
+from api.schemas import CityListSchema, CityCreateSchema, CityUpdateSchema, TemperatureListSchema
+
+router = APIRouter()
+
+
+@router.get("/cities", response_model=List[CityListSchema])
+def list_cities(db: Session = Depends(get_db)):
+    return get_cities(db=db)
```

Comment:
```suggestion
@router.get("/cities/", response_model=List[CityListSchema])
def list_cities(db: Session = Depends(get_db)):
    return get_cities(db=db)
```

Please fix everywhere